doc/qbk/02_2_strings.qbk

```diff
@@ -12,22 +13,58 @@
 [section:strings Using Strings]
 
 Modifiable sequences of characters are represented using instances of type
-[link json.ref.boost__json__string `string`].
-These strings have an interface almost identical to `std::string`, and
-operate the same way, with these variations:
+__string__.
+These strings have an modified `string_view` based version of the `std::string` interface, and
+operate the same way, except that:
 
-* __string__ is not a class template.
+* __string__ is not a class template,
+
+* __string__ only uses `char` as its character type,
+
+* redundant overloads for string operations have been replaced with a `string_view` based interface,
+
+* access to characters in the range `[size(), capacity()]` is permitted, and
 
 * __storage__ is used instead of __Allocator__.
 
-* New characters are 
+With the `string_view` based interface, operations requiring an input string are implemented as a single overload with
+a parameter of type `string_view`, and can accept most string-like objects in addition to other `string_view`s. 
```

Comment:
You're using the word string and string_view way too many times it is like "string string and string string view string the string view the viewed string view stringed view" - blah